posthog/models/organization.py

```diff
@@ -176,34 +186,37 @@ def _billing_plan_details(self) -> tuple[Optional[str], Optional[str]]:
                 return (license.plan, "ee")
         return (None, None)
 
-    def update_available_features(self) -> list[Union[AvailableFeature, str]]:
-        """Updates field `available_features`. Does not `save()`."""
+    def update_available_product_features(self) -> list[ProductFeature]:
+        """Updates field `available_product_features`. Does not `save()`."""
         if is_cloud() or self.usage:
-            # Since billing V2 we just use the available features which are updated when the billing service is called
-            return self.available_features
+            # Since billing V2 we just use the field which is updated when the billing service is called
+            return self.available_product_features  # type: ignore
```

Comment:
Why do you need the ignore now but you didn't before?

Reply:
The typing isn't particularly friendly bc of the database field allowing nulls so I've updated this to be `return self.available_product_features or []`, which also silences the error. The problem before was that mypy wasn't happy with the possibility of the function returning None when it's signature is labelled `list[ProductFeature]`